powerquery-docs/connectors/celonis-ems.md

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
This is also missing the changes to the index.md file and the ToC

Reply:
Currently waiting for the logo - for the index.md file.